src/carousel.tsx

```diff
@@ -62,22 +63,54 @@ export const PageBullets: React.FC<PageBulletsProps> = ({currentIndex, numPages,
     );
 };
 
-type ItemsPerPageProp = {mobile?: number; desktop?: number} | number;
+type DesktopItemsPerPage = {small?: number; medium?: number; large?: number} | number;
+type ItemsPerPageProp = {mobile?: number; tablet?: number; desktop?: DesktopItemsPerPage} | number;
```

Comment:
Do you think we'll need the different values for desktop? I'd say we'll always want `itemsPerPage.desktop.small` to be equal to `itemsPerPage.mobile` or `itemsPerPage.desktop.medium` to be equal to `itemsPerPage.tablet`.

If we asume that, we can simplify the api surface of the component

Reply:
As talked with design, they want different values :)

Reply:
I can't remember talk about that @wyunreal 🤔 

In a sidebar on desktop, you can use mobile itemsPerPage by default, I can't find any case against this. Do you know any example?

Maybe we need to include this detail in specs to avoid confussions, @aweell ?

Reply:
![image](https://user-images.githubusercontent.com/7761987/208082244-e3eba580-6a5d-406c-a72e-227b13c3b5c5.png)

Reply:
Talk is not a talk, its a figma
Also, I think its a really bad idea to think that a small column on desktop will be always the same than  a mobile device, same with mid size desktop column and tablet, its a bad idea to couple these different concepts.

(and we already have a requirement to have this different cases).

Reply:
figma: https://www.figma.com/file/xNUwhH29z3Fre4foNltgtX/CPS_Specs?node-id=2670%3A59287&t=oRYoD9xihBs6ewJB-0

Reply:
> In a sidebar on desktop, you can use mobile itemsPerPage by default, I can't find any case against this. Do you know any example?

By default, we are using this as you say.

Reply:
> ![image](https://user-images.githubusercontent.com/7761987/208082244-e3eba580-6a5d-406c-a72e-227b13c3b5c5.png)

`2.2` ? we shouldn't use decimals here. Appart from that, I see the equivalence between mobile and small column and tablet and medium column

Reply:
I dont see the equivalence, are totally different concepts. but I habe a requirement on this figma, if you think its better to wait, I suggest @yceballost to talk to @MontseTelefonica  in order to clarify this. If designers thinks we can just have mobile / desktop / tablet, I'm ok, but please think it twise, a small column on desktop is not a mobile device and viceversa ( think that there are more than one small column sizes because it depends on the grid template the page is using, same with medium column sizes).

Reply:
@pladaria wdyt ? do you think a mobile or tablet is the same, conceptually, than a small / medium desktop column ? do you think its semantic to say: mobile: 2 and expects to have 2 also in small desktop column ? TBH, I think its a bad API design buttt, if you think its a good API design, I will follow your recommendation for sure :)

Reply:
With "I see the equivalence" I meant in that concrete definition (except for the decimal part):
- mobile: 1 -> desktop-small-column: 1
- tablet 2.2 -> desktop-medium-column: 2

This isn't an API design discussion, it should be a design needs discussion. If Design doesn't need the extra configurability I wouldn't include it in the component. If it's really needed, then I'm ok

Reply:
Its needed as we have a Figma requiring it ;)

Reply:
So can you please approve the PR ? or at least please move the question to design :)

Reply:
Ok, talked in a call with Design. This is a requirement, so the api makes sense